App/Containers/Comment.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
```

Comment:
This UI to enter a photo comment isn't being used yet, just leaving it in place for when we're ready.